test/validations.test.js

```diff
@@ -322,6 +322,47 @@ describe('validations', function() {
       });
     });
 
+    it('should pass options to custom synch', function(done) {
+      delete User.validations;
+      User.validate('name', function(err, options) {
+        if (options.testFlag !== 'someValue') err();
+      });
+      User.create({name: 'Valid'}, {testFlag: 'someValue'}, function(e, d) {
+        d.updateAttribute('name', null, {testFlag: 'otherValue'}, function(e) {
+          should.exist(e);
+          e.should.be.instanceOf(ValidationError);
+          d.updateAttribute('name', 'Vasiliy', {testFlag: 'someValue'}, function(e) {
+            should.not.exist(e);
```

Comment:
```js
if (err) return done(err);
```

Reply:
The code pattern here is just copied from all the other validation tests above.  The specific case we're concerned with is that when we pass the correct "testFlag" value we either do or do not get an exception when we do the updateAttribute.  In writing the test case it's clearer to state that we are expecting a specific exception, or we are expecting NO exception, which is what this does.  

Put another way, the "test failure" in this case is not the content of the exception, it's either its presence or absence.